docs/griptape-framework/misc/events.md

```diff
@@ -74,6 +74,18 @@ Handler 1 <class 'griptape.events.finish_structure_run_event.FinishStructureRunE
 Handler 2 <class 'griptape.events.finish_structure_run_event.FinishStructureRunEvent'>
 ```
 
+## Stream Iterator
+
+You can use `Structure.run_stream()` for streaming Events from the `Structure` in the form of an iterator.
+
+!!! tip
+
+Set `stream=True` on your [Prompt Driver](../drivers/prompt-drivers.md) in order to receive completion chunk events.
```

Comment:
I would love to see this requirement be inferred based on Structure usage rather than needing it to be set in the constructor. Obviously out of scope, but writing it down until I have time to fill out an issue.

Reply:
Agreed, we need a better way for framework components to broadcast that they publish stream-like events.